src/helpers/common/components/ValidSectionRenderer.tsx

```diff
@@ -0,0 +1,30 @@
+import { Fragment, ReactNode, useMemo } from 'react';
+import { IExperienceItem } from 'src/stores/experience.interface';
+import { IEducationItem } from 'src/stores/education.interface';
+import { IAwardItem } from 'src/stores/awards.interface';
+import { IVolunteeringItem } from 'src/stores/volunteering.interface';
+import { ISkillItem } from 'src/stores/skill.interface';
+
+export const ValidSectionRenderer = ({
```

Comment:
```suggestion
export const SectionValidator = ({
```
is it conveying @siva-kannan3 ?

Reply:
yeah